.github/workflows/reuse-store-image-name-and-tags.yml

```diff
@@ -30,7 +30,12 @@ jobs:
       - name: Compute version tags
         id: step1
         run: |
-          echo COMMIT_TAG=$(git rev-parse --short "$GITHUB_SHA") >> $GITHUB_OUTPUT
+          # For PR, GITHUB_SHA is NOT the last commit pushed onto the PR branch - https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+            echo "COMMIT_TAG=$(git rev-parse --short ${{ github.event.pull_request.head.sha }})" >> $GITHUB_OUTPUT
```

Comment:
I assume this will return the head commit hash of the feature branch instead of the head of the temp merge branch, right?

Reply:
yep that's correct 👍